ga4gh/backend.py

```diff
@@ -286,6 +292,18 @@ def startProfile(self):
     def endProfile(self):
         pass
 
+    def validateRequest(self, jsonDict, requestClass):
+        if ('REQUEST_VALIDATION' in flaskapp.app.config and
+                flaskapp.app.config['REQUEST_VALIDATION']):
+            if not requestClass.validate(jsonDict):
+                raise backendExceptions.RequestValidationFailureException
```

Comment:
Yeah, I think we should. It's just making an awkward special case out of no-parameter exceptions.